docs/Versioning.md

```diff
@@ -127,6 +127,24 @@ Notes:
 - Values that are new or updated from a previous OpSet version are in **bold**.
 - \*: the operator is deprecated.
 
+### Operator Deprecations
```

Comment:
Consider adding this info to https://github.com/onnx/onnx/blob/5ca0a09e6846fe70c9dda8b5034f4a1454c6d130/docs/ManagingExperimentalOps.md and linking to it instead. Experimental ops do not have any versioning and now the concept itself is deprecated, so it's odd to list them in the mainline documentation.